### ISSUE TITLE
How to receive array with selected seats for checkout?

### ISSUE THREAD

**Issue by AllRaDiO** [mateuszmarkowski/jQuery-Seat-Charts#37]
Thу main question is how to receive array with selected seats?

**Comment by computerwizjared** [comment]
I do this by adding and removing from an array during the "click" handler within the seatCharts function. For an example, look here: https://github.com/XiTickets/TheatreTickets/blob/master/public/js/tickets.js#L231

**Comment by AllRaDiO** [comment]
Yeah, that's it. It would be great to add this information to demo.

**Comment by kirantpatil** [comment]
Hi, May I know where is the demo code ?
